pynamodb/models.py

```diff
@@ -141,7 +142,6 @@ class DefaultMeta(object):
     region = DEFAULT_REGION
     host = None
 
-
```

Comment:
do not delete these lines, PEP8 is 2 blank lines between classes https://www.python.org/dev/peps/pep-0008/#blank-lines